[refresh2020q1][img] - go error while building img

This error is occuring during the refresh process. I have posted the context of the error and linked the file at the bottom of the issue.

Error:

```
mkdir -p /hab/pkgs/core/img/0.5.4/20200330181137/cross/bin
mv /hab/pkgs/core/img/0.5.4/20200330181137/cross/src/github.com/opencontainers/runc/runc /hab/pkgs/core/img/0.5.4/20200330181137/cross/bin/runc
go: github.com/containerd/containerd@v1.3.0-0.20190426060238-3a3f0aac8819: invalid pseudo-version: version before v1.3.0 would have negative patch number
make: *** [Makefile:28: internal/binutils/runc.go] Error 1
   img: Build time: 0m43s
   img: Exiting on error
```

Steps to reproduce:

```
cd /tmp/
git clone --depth 1 -b refresh/2020q1_008 --single-branch https://github.com/habitat-sh/core-plans.git /tmp/core-plans"
wget https://gist.githubusercontent.com/MindNumbing/255ca89882213bf0a4422f28fb902570/raw/b00ed3638b73d8a697ca411ea5154bd77118d456/hab_studio.env
source hab.env
hab studio enter
build core-plans/img
```

Full Error Log:
[img.txt](https://github.com/habitat-sh/core-plans/files/4458545/img.txt)

--- a/.refresh/remaining_plans_failed.txt
+++ b/.refresh/remaining_plans_failed.txt
@@ -7,7 +7,6 @@ core-plans/buildkite-agent
 core-plans/ansible
 core-plans/shield-agent
 core-plans/rethinkdb
-core-plans/img
 core-plans/cockroach
 core-plans/buildkite-cli
 core-plans/drupal

--- a/.refresh/remaining_plans_passed.txt
+++ b/.refresh/remaining_plans_passed.txt
@@ -319,6 +319,7 @@ core-plans/libpciaccess
 core-plans/kubectl
 core-plans/kibana
 core-plans/intltool
+core-plans/img
 core-plans/imagemagick
 core-plans/renderproto
 core-plans/kbproto

--- a/img/plan.sh
+++ b/img/plan.sh
@@ -1,6 +1,6 @@
 pkg_name="img"
 pkg_origin="core"
-pkg_version=0.5.4
+pkg_version=0.5.7
 pkg_description="Standalone, daemon-less, unprivileged Dockerfile and OCI compatible container image builder."
 pkg_upstream_url="https://github.com/genuinetools/img"
 pkg_license=('MIT')
@@ -39,6 +39,14 @@ do_verify() {
 }
 
 do_unpack() {
+  return 0
+}
+
+do_prepare() {
+  pushd "${pkg_cache_path}" >/dev/null
+    go mod edit -replace github.com/containerd/containerd@3a3f0aac8819=github.com/containerd/containerd@814b795
+    go get github.com/go-bindata/go-bindata/go-bindata
+  popd >/dev/null
   return 0
 }
 

--- a/img/plan.sh
+++ b/img/plan.sh
@@ -43,11 +43,12 @@ do_unpack() {
 }
 
 do_prepare() {
+  # This changes the version of containerd that is added to fix invalid pseudo version caused by version in go.mod file.
+  # This should be fixed with the next release.
   pushd "${pkg_cache_path}" >/dev/null
     go mod edit -replace github.com/containerd/containerd@3a3f0aac8819=github.com/containerd/containerd@814b795
     go get github.com/go-bindata/go-bindata/go-bindata
   popd >/dev/null
-  return 0
 }
 
 do_build() {

